misc/docker-ci/check.mk

```diff
@@ -35,6 +36,13 @@ ossl1.1.1:
 		BUILD_ARGS='$(BUILD_ARGS)' \
 		TEST_ENV='$(TEST_ENV)'
 
+ossl3.0:
+	docker run $(DOCKER_RUN_OPTS) $(CONTAINER_NAME) \
+		make -f $(SRC_DIR).ro/misc/docker-ci/check.mk _build_ossl3.0 _check \
+		CMAKE_ENV='PKG_CONFIG_PATH=/opt/openssl-3.0/lib64/pkgconfig' \
```

Comment:
CMake available in Ubuntu 20.04 cannot detect OpenSSL 3.0 when specified by `-DOPENSSL_ROOT_DIR`. Hence use `PKG_CONFIG_PATH`.

However, doing this only does not fix the build issue, because picotls-cli (a dependency of `make checkdepends`) takes the OpenSSL path being resolved and passes it to CMake of picotls using `-DOPENSSL_ROOT_DIR`.

ToDo somehow fix this problem.